app/code/Magento/SendFriend/Model/SendFriend.php

```diff
@@ -372,6 +372,20 @@ public function setSender($sender)
         return $this->setData('_sender', new \Magento\Framework\DataObject($sender));
     }
 
+    /**
+     * Set the reply to email address
+     *
+     * @param array $sender
+     * @return void
+     */
+    public function setReplyTo($sender)
```

Comment:
Unfortunately, we are not able to add new public methods to the @api classes according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)